rstcheck.py

```diff
@@ -379,24 +341,9 @@ def unknown_departure(self, node):
 
 def beginning_of_code_block(node, full_contents):
     """Return line number of beginning of code block."""
-    lines = full_contents.splitlines()
     line_number = node.line
-    code_block_length = len(node.rawsource.splitlines())
```

Comment:
Now for the validation code used (sphinx.directives.code.CodeBlock). He returns for another line number, I suspect that because of this function https://github.com/sphinx-doc/sphinx/blob/master/sphinx/directives/code.py#L125